app/styles/_modules.scss

```diff
@@ -1,3 +1,4 @@
+@import '../bower_components/jquery-modal/jquery.modal.css';
```

Comment:
This is kind of gross answer, but I don't think it will work.
Based on my quick reading and superficial understanding of https://github.com/sass/sass/issues/556 it doesn't look like you can `@import` a vanilla .css file. I did a quick test and it seems that it worked as expected if I renamed the file to `jquery.modal.scss` (.scss vs .css).

Not sure if we can just include the jquery.modal.css in the HTML file, similar to how we're bundling the `normalize.css` (see [/server/templates/pages/src/index.html:12](https://github.com/mozilla/fxa-content-server/blob/4cade11b2eeb211aac7a84cc2f2fb9cb1b0f002b/server/templates/pages/src/index.html#L12))

... or, we have a Grunt task that renames the `jquery.modal.css` -> `jquery.modal.scss` and stick w/ `@import`.

I'll continue to ~~Google~~ Yahoo! around for a different/better solution.